admin-print-action/src/PrintActionExtension.liquid

```diff
@@ -158,6 +175,16 @@ extend(TARGET, (root, { i18n, data }) => {
 
   root.append(printActionComponent);
 
+  /* 
+    This template fetches static documents from the CDN to demonstrate printing.
+    However, when building your extension, you should update the src document
+    to match the resource that the user is printing. You can do this by getting the 
+    resource id from the data API and using it to create a URL with a path to your app 
+    that shows the correct document. For example, you might use a URL parameter to 
+    render an invoice for a specific order.
+    
+    https://www.example.org/print/invoice&orderId={data.selected}
```

Comment:
Same change as above

Also, since we support relative URLs, we could also just leave off the example.org bit.